packages/components/src/toolbar-button/index.js

```diff
@@ -28,48 +23,42 @@ function ToolbarButton( {
 	isDisabled,
 	extraProps,
 	children,
+	...props
 } ) {
-	// It'll contain state if `ToolbarButton` is being used within
-	// `<Toolbar __experimentalAccessibilityLabel="label" />`
-	const accessibleToolbarState = useContext( ToolbarContext );
-
-	const button = (
-		<IconButton
-			icon={ icon }
-			label={ title }
-			shortcut={ shortcut }
-			data-subscript={ subscript }
-			onClick={ ( event ) => {
-				event.stopPropagation();
-				if ( onClick ) {
-					onClick( event );
-				}
-			} }
-			className={ classnames(
-				'components-toolbar__control',
-				className,
-				{ 'is-active': isActive }
-			) }
-			aria-pressed={ isActive }
-			disabled={ isDisabled }
-			{ ...extraProps }
-		/>
-	);
-
-	if ( accessibleToolbarState ) {
+	if ( icon ) {
 		return (
-			<AccessibleToolbarButtonContainer className={ containerClassName }>
-				{ button }
-			</AccessibleToolbarButtonContainer>
+			<ToolbarButtonContainer className={ containerClassName }>
+				<IconButton
+					icon={ icon }
+					label={ title }
+					shortcut={ shortcut }
+					data-subscript={ subscript }
+					onClick={ ( event ) => {
+						event.stopPropagation();
+						if ( onClick ) {
+							onClick( event );
+						}
+					} }
+					className={ classnames( 'components-toolbar__control', className, {
+						'is-active': isActive,
+					} ) }
+					aria-pressed={ isActive }
```

Comment:
Should `isActive` be a state on `IconButton` instead of toolbar icon button?

Reply:
See https://github.com/WordPress/gutenberg/pull/17748. I want to make `isPressed` (it's still `isToggled` but we need a different name) part of the `Button` component. In effect, `IconButton` would inherit it.

Reply:
In the similar vein, I'd like to replicate `focusable` and `disabled` fair implemented in Reakit for `Button` :)

I don't like the fact that we use ARIA attributes as of today.

Reply:
Cool! So, I'll keep that unchanged on this PR.